### PR TITLE
Added home work for 2. JS Basics

### PR DESCRIPTION
**Pull request by AlinaDashko**

### CODE REVIEW

File: js_basics/task5-2.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const length = 40;
+const width = 20;
+const sum = length * width;
```
Comment (sttaran) [inline js_basics/task5-2.js]: not sum but area of rectangle

File: js_basics/task5-3.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const length = 40;
+const width = 20;
+const sum = length * width;
```
Comment (sttaran) [inline js_basics/task5-3.js]: duplicated code, you have the same in previous file

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_basics/task5-3.js (hunk 2 above)
Comment (AlinaDashko) [reply js_basics/task5-3.js]: Thank you! I made edits for variable name for task5-2.js , changed duplicated code for task5-3.js .

File: js_basics/task5-3.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+let length = 25;
```
Comment (sttaran) [inline js_basics/task5-3.js]: why do you still have it here? You have solved this task in `js_basics/task5-2.js ` file.

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_basics/task5-3.js (hunk 3 above)
Comment (AlinaDashko) [reply js_basics/task5-3.js]: I get it! For some reason, I thought that I should also display the area in task5-3.js 
I also wondered why I should print it a second time. 
Thanks, I fixed it.

**Pull request merged**